packages/admin/admin/src/fullPageAlert/FullPageAlert.styles.ts

```diff
@@ -46,12 +46,12 @@ export const ContentContainer = createComponentSlot(Card)<FullPageAlertClassKey>
 export const Title = createComponentSlot(Typography)<FullPageAlertClassKey>({
     componentName: "FullPageAlert",
     slotName: "title",
-})(css``);
+})() as typeof Typography;
```

Comment:
Why do we need this as cast?

Reply:
when moving to the slot approach, this as cast is not needed anymore.